### PR TITLE
Power Ups - phase 1

### PR DESCRIPTION
**Pull request by RumiaTouhou**

### CODE REVIEW

File: core/src/com/mygdx/game/PirateGame.java [hunk 1]
```diff
@@ -6,7 +6,12 @@
 import com.badlogic.gdx.scenes.scene2d.ui.Skin;
 import com.badlogic.gdx.utils.viewport.ScreenViewport;
 import com.mygdx.game.Managers.ResourceManager;
-import com.mygdx.game.UI.*;
```
Comment (printincident) [inline core/src/com/mygdx/game/PirateGame.java:9]: Put this line back as we just want to import everything

File: core/src/com/mygdx/game/Components/Pirate.java [hunk 2]
```diff
@@ -13,11 +13,15 @@
  */
 public class Pirate extends Component {
     private int factionId;
-    public static int plunder;
```
Comment (ameliarwig) [inline core/src/com/mygdx/game/Components/Pirate.java:16]: why no more static?

File: core/src/com/mygdx/game/PirateGame.java [hunk 3]
```diff
@@ -6,7 +6,12 @@
...
+import com.mygdx.game.UI.EndScreen;
+import com.mygdx.game.UI.GameScreen;
+import com.mygdx.game.UI.MenuScreen;
+import com.mygdx.game.UI.ShopScreen;
+
+import javax.annotation                                                                                                                                                        .Resource;
```
Comment (printincident) [inline core/src/com/mygdx/game/PirateGame.java:14]: I don't even know what this is

Comment (ameliarwig) [inline core/src/com/mygdx/game/PirateGame.java:14]: ??? red ting

File: core/src/META-INF/MANIFEST.MF [hunk 4]
```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```
Comment (printincident) [inline core/src/META-INF/MANIFEST.MF:1]: Why is there a new Manifest

File: core/src/com/mygdx/game/Components/Pirate.java [hunk 5]
```diff
@@ -13,11 +13,15 @@
...
+    private int plunder;
     protected boolean isAlive;
-    public static int health;
-    public static int ammo;
-    private final int attackDmg;
+    private boolean isImmortality;
```
Comment (printincident) [inline core/src/com/mygdx/game/Components/Pirate.java:18]: Could this not be managed by a object

File: core/src/com/mygdx/game/Entitys/TimeBar.java [hunk 6]
```diff
@@ -0,0 +1,27 @@
+package com.mygdx.game.Entitys;
+
+import com.badlogic.gdx.math.Vector2;
+import com.mygdx.game.Components.Renderable;
+import com.mygdx.game.Components.Transform;
+import com.mygdx.game.Managers.RenderLayer;
+import com.mygdx.game.Managers.ResourceManager;
+
+/**
+ * @author Rrryaan
```
Comment (printincident) [inline core/src/com/mygdx/game/Entitys/TimeBar.java:10]: Where did this come from

**Review by printincident — CHANGES_REQUESTED** [review]

File: core/src/com/mygdx/game/PirateGame.java (hunk 3 above)
Comment (printincident) [reply core/src/com/mygdx/game/PirateGame.java:14]: does this even compile??

Comment (printincident) [reply core/src/com/mygdx/game/PirateGame.java:14]: it doesn't

**Comment by dom-hall** [comment]
There are a lot of changes to existing code and deleted work by other team members. This is being resolved in PR #33.

**Pull request closed without merging**